test/system/place_tags_test.rb

```diff
@@ -0,0 +1,20 @@
+require "application_system_test_case"
+
+class PlaceTagsTest < ApplicationSystemTestCase
+
+  # node blog has lat and lon tag and place tag .	
+  test "pages tagged with place get a full-width map across the top of the page" do
```

Comment:
Love this!!!

Reply:
Hey, @sagarpreet-chadha can you see if I'm doing anything wrong here? https://github.com/publiclab/plots2/pull/5605/